pages/services/spark/2.3.1-2.2.1-2/hdfs/index.md

```diff
@@ -24,41 +27,58 @@ Within the {{ model.techShortName }} service configuration, set `hdfs.config-url
   }
 }
 ```
-This can also be done through the web interface. If you are using the default installation of HDFS from Mesosphere, this is probably `http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints`.
+This can also be done through the web interface. If you are using the default installation of HDFS from Mesosphere, this is probably http://api.hdfs.marathon.l4lb.thisdcos.directory/v1/endpoints.
 
 <a name="adding-hdfs"></a>
+
 ## Adding HDFS configuration files per-job
 To add the configuration files manually for a job, use `--conf {{ model.serviceName }}.mesos.uris=<location_of_hdfs-site.xml>,<location_of_core-site.xml>`. This will download the files to the sandbox of the Driver {{ model.techShortName }} application, and DC/OS {{ model.techName}} will automatically load these files into the correct location. 
 
-**Note:** It is important these files are called `hdfs-site.xml` and `core-site.xml`.
+<table class=“table” bgcolor=#858585>
+<tr> 
+  <td align=justify style=color:white><strong>Important:</strong> It is important that these files are called "hdfs-site.xml" and "core-site.xml".</td> 
```

Comment:
small nit: can we change message to `It is important these files are named "hdfs-site.xml" and "core-site.xml"`?